src/views/PeopleSearch/index.js

```diff
@@ -461,14 +498,28 @@ class PeopleSearch extends Component {
   handleEnterKeyPress = (event) => {
     if (event.key === 'Enter') {
       this.search();
-      this.updateURL();
     }
   };
 
   render() {
     const { classes } = this.props;
     let PeopleSearchCheckbox;
 
+    const printPeopleSearchHeader = (
+      <div class="test" align="center" style={{ display: 'none' }}>
```

Comment:
"class" wasn't a valid DOM property, but this code is in develop now.

Reply:
This is actually a JSX thing, not Material UI (as can be seen by the fact that `<div>` is not a MUI component). The issue is that `class` is a reserved keyword in JavaScript (for creating object-oriented classes). Since JSX is a syntactic sugar over JavaScript, it couldn't use the `class` keyword, so it uses `className` for declaring the classes of a JSX component.